### PR TITLE
Switch CouchDB with cloudant-python-sdk and get version from git release

### PR DESCRIPTION
**Pull request by aanil**
Using cloudant-python-sdk seems to solve the issue with sending large json files to couch.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,12 +9,15 @@
...
+
+def get_version():
+    return subprocess.check_output(["git", "describe", "--tags"]).strip().decode("utf-8")
 
 setup(
     name = 'multiqc_ngi',
-    version = version,
+    version = get_version(),
     author = 'Phil Ewels',
     author_email = 'phil.ewels@scilifelab.se',
```
Comment (FranBonath) [inline setup.py]: this email is meaningless, since it is inactive now.

**Review by FranBonath — APPROVED** [review]
Just double checking, you want this to be merged to "master", right?

File: setup.py (hunk 1 above)
Comment (aanil) [reply setup.py]: Removed author from setup now

**Review by alneberg — APPROVED** [review]
Looks good!

**Pull request merged**